shared/protos/Room.proto

```diff
@@ -107,6 +107,17 @@ message Room {
 
   repeated Tile tiles = 18 [(gmx) = "tiles/tile"];
 
+  message Layer {
+    optional int32 depth = 1;
+    optional string name = 2;
```

Comment:
I feel like name should be the first field.

Reply:
Good idea, I also forgot the layer has an id too, which I've added, but I don't know what they start at yet.